src/hooks/series/useEpisodes.ts

```diff
@@ -0,0 +1,95 @@
+import { UseInfiniteQueryResult, useInfiniteQuery } from 'react-query';
+
+import { getEpisodes, getSeasonWithEpisodes } from '#src/services/api.service';
+import type { EpisodesWithPagination } from '#types/series';
+import type { ApiError } from '#src/utils/api';
+
+const CACHE_TIME = 60 * 1000 * 60;
+
+// Get episodes from a selected series using pagination
+export const useSeriesEpisodes = (seriesId: string | undefined, isNewSeriesFlow: boolean): UseInfiniteQueryResult<EpisodesWithPagination, ApiError | null> => {
```

Comment:
I think this will make the code more understandable and also not rely on the understanding of what 'new series flow' means.

```suggestion
export const useSeriesEpisodes = (seriesId: string | undefined, options?: {enabled?: boolean}): UseInfiniteQueryResult<EpisodesWithPagination, ApiError | null> => {
```

Reply:
I agree, thanks!